src/main/java/uk/gov/hmcts/reform/userprofileapi/resource/UpdateUserProfileData.java

```diff
@@ -15,10 +17,17 @@
 @Setter
 public class UpdateUserProfileData implements RequestData {
 
+    @JsonIgnore
+    private static final String NAME_FORMAT_REGEX = "^[A-Za-z'-]+$";
+    @JsonIgnore
```

Comment:
remove json ignore from here and move the entry to ocnstant file

Reply:
This has been addressed in this commit: https://github.com/hmcts/rd-user-profile-api/pull/257/commits/cae47ccd2202a8e751f5e6f2ccf79d7c62b5f921